server/organization/views.py

```diff
@@ -14,3 +16,28 @@ class OrganizationViewSet(viewsets.ModelViewSet):
     queryset = Organization.objects.all()
     serializer_class = OrganizationSerializer
     permission_classes = [IsAuthenticated, UserOrganizationPermission]
+
+
+class ModifyOrganizationInventoryItemsDataUpdate(generics.GenericAPIView):
+    """
+    API endpoint that allow a user to update the timing at which
+    the Inventory Data is refreshed
+    """
+    permission_classes = [IsAuthenticated]
```

Comment:
you should update the permission to be more restrictive

Reply:
Okay I wanted to discuss that. So the restrictions should be SA and IM that are just part of the organization right?

Reply:
Yep